src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -2370,4 +2446,25 @@ private class RD2_DataMigration_TEST {
         return Database.insert(new Opportunity(), false);
     }
 
+    /***
+    * @description Updates first two opportunities in the recurring donation to closed won for calculating rollups
+    */
+    private static void setRdOppsToClosedWonForRollupTests(npe03__Recurring_Donation__c rd, Integer numOpps) {
+        List<Opportunity> oppToUpdate = new List<Opportunity>();
+        List<Opportunity> oppList =  oppGateway.getRecordsByCloseDate(rd);
```

Comment:
Data type such as List, Map, Set are not required in the names since methods are short and we can see what each var data type is and how it can be used, so we can just call this var `opps` or `rdOpps` etc.